### ISSUE TITLE
Code Release ? 

### ISSUE THREAD

**Issue by AMEERAZAM08** [snap-research/InstantRestore#1]

**Comment by dawei03896** [comment]
+1

**Comment by 5agado** [comment]
Any updates regarding the code release?

**Comment by JackVinati** [comment]
they will never release it like 90% of the papers outhere

**Comment by MentalGear** [comment]
Yeah, it was a PR move to put it on github  On 18 May 2025, at 14:16, Giacomo Vinati ***@***.***> wrote:﻿
JackVinati left a comment (snap-research/InstantRestore#1)
they will never release it like 90% of the papers outhere

**Comment by oliverban** [comment]
> they will never release it like 90% of the papers outhere

Well, well, well
